go.mod

```diff
@@ -88,7 +88,7 @@ require (
 	github.com/cli/safeexec v1.0.1
 	github.com/coder/flog v1.1.0
 	github.com/coder/pretty v0.0.0-20230908205945-e89ba86370e0
-	github.com/coder/quartz v0.1.0
+	github.com/coder/quartz v0.1.2
```

Comment:
Interestingly, the backpressure test surfaced some bugs in Quartz that I had to fix to get the mock to behave like the real TickerFunc.